docs/sources/panels-visualizations/visualizations/pie-chart/index.md

```diff
@@ -35,6 +35,21 @@ The following video guides you through the creation steps and common customizati
 
 {{< docs/play title="Grafana Bar Charts and Pie Charts" url="https://play.grafana.org/d/ktMs4D6Mk/" >}}
 
+## Supported data formats
+
+The pie chart visualization requires a dataset with at least one row of numeric values to be added together. If the first row contains column titles, it's used to display and identify the values in the pie chart. Cells with non-numeric values are not displayed in the pie chart.
+
+Example:
+
+| Description   | Internet Explorer | Chrome |
+| ------------- | ----------------- | ------ |
+| Other users   | 1.5               | 7      |
+| English users | 17.5              | 50     |
```

Comment:
I wonder if it makes sense to start with a simpler example where all the content will show up in the pie chart and then add this as a second example. And explain what will and will not appear in the chart in each example. For example:

> | Internet Explorer | Chrome |
> ----------------------------
> | 17  | 50 |
> 
> In this example you'll get a pie chart that looks like this (add image). 

I also wonder if you can add information about use cases for a pie chart; what kind of data you'd want to display this way. The dataset in the example doesn't seem like the type of thing you'd plug into a pie chart because you're trying to show multiple dimensions, I think, but I'm not sure.

Also, how do you have a half a user?